ch(setup tests):#2

Shumba250 opened the pull request:
#### What does this PR do?

- passing test sample
- a test environment is a setup and a mirror database is created
- a coverage badge added to the readme.md file

#### Description of Task to be completed

- a sample test case passing and automatically switched to the test database when testing
- a coverage badge is added to the readme.md after the tests are run
- created a mirror test database

#### How should this be manually tested?

- clone the repo
- type npm install in the terminal to install the necessary packages
- to run the tests type npm test

Shumba250 committed:
ch(setup tests):
a sample test case passing
automatically changes to test database when running tests
setup a test environment
test coverage badge added to a readme.md file

wakaflorien reviewed (changes requested):
- add trello card number in your description.
- i don't get whay you are force exiting.
- correct these errors in exiting.

![image](https://user-images.githubusercontent.com/48131185/222649044-87925271-030b-44f6-bb4a-14c6a2319a9e.png)

Shumba250 committed:
ch(setup tests):
a sample test case passing
automatically changes to test database when running tests
setup a test environment
test coverage badge added to a readme.md file

sostene12 reviewed (changes requested):
i think it can be better if you don't push coverage folder

Shumba250 committed:
ch(setup tests):
a sample test case passing
automatically changes to test database when running tests
setup a test environment
test coverage badge added to a readme.md file

Shumba250 committed:
ch(setup tests):
a sample test case passing
automatically changes to test database when running tests
setup a test environment
test coverage badge added to a readme.md file

Shumba250 committed:
ch(setup tests):
a sample test case passing
automatically changes to test database when running tests
setup a test environment
test coverage badge added to a readme.md file
